src/components/storage-browser.js

```diff
@@ -732,6 +747,8 @@ export default function StorageBrowserWrapper({ ...props }) {
       user={user}
       storage={storage}
       setStorage={setStorage}
+      getUploadLiabilityCookie={() => getCookie(uploadLiabilityCookieName)}
+      setUploadLiabilityCookie={() => setLongLastingCookie(uploadLiabilityCookieName, 'true')}
```

Comment:
I think this should be `setSessionCookie` if we want it to reappear after the browser is re-opened?